sources/engine/Stride.Engine/Profiling/DebugTextSystem.cs

```diff
@@ -17,10 +17,11 @@ internal struct DebugOverlayMessage
             public string Message;
             public Int2 Position;
             public Color4 TextColor;
+            public TimeSpan RemeaningTime;
```

Comment:
Typo, should be `RemainingTime` :)

Reply:
Sorry about it, and thank you for pointing it out to me so i can fix it.